### PR TITLE
add option "php composer.phar"

### PR DESCRIPTION
**Pull request by franzholz**
If you install composer, then you only get the composer,.phar executable file. A command 'composer' is not present.

### CODE REVIEW

**Comment by aik099** [comment]
If you install it globally, then command will be `composer`. I guess you're talking about local install.

**Comment by franzholz** [comment]
Yes, I have read the docu from composer. I did only follow those steps on the composer website. I did not come to the point of a global install.

**Comment by stof** [comment]
Can you add it inside a note, and mention that all other examples in the doc (add in #30 and #31) will only show the command in global mode ?

On a side note, the Symfony community decided to use the global command everywhere in its doc (and many third party Symfony bundles have been updated to follow the same rule). There is a discussion in the composer repo about showing the global install first in their doc btw.

**Comment by stof** [comment]
@franzholz maintainers are not notified when a new commit is pushed to a PR, so it is a good idea to leave a comment to notify us. Otherwise PRs might get lost (I saw the new commit only because I decided to review the pending again before working on the doc on my side)

File: index.rst [hunk 1]
```diff
@@ -27,6 +27,10 @@ The recommended way to install Mink with all its dependencies is through
 
     $ composer require behat/mink
 
+.. note::
+    For local installations of composer you must call it like this: ``$ php composer.phar require behat/mink`` .
```
Comment (stof) [inline index.rst]: Can you add an empty line between the ``.. note::`` directive and its content and wrap the content after the first word crossing 72 chars to keep the lines shorter in the source ?

@aik099 FYI, this CS rule about the wrapping comes from the Symfony documentation standards. I decided to apply the same standards

Comment (aik099) [inline index.rst]: > @aik099 FYI, this CS rule about the wrapping comes from the Symfony documentation standards. I decided to apply the same standards

Thanks god, that you're not considering 85 symbols per line limit from PEAR standard in PHP. Generally I'm against short lines and think that 120 symbols per line is just proper line length limit.

**Pull request merged**